docs/content/opentracing.md

```diff
@@ -0,0 +1,11 @@
+---
+title: OpenTracing Support
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
+This doc is now available at https://docs.nginx.com/nginx-ingress-controller/third-party-modules/opentracing/
```

Comment:
delete this doc and add an alias in the new file's frontmatter

Reply:
done